src/components/home.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component, PropTypes } from 'react';
+
+const BASE_URL = "https://dev-570863.oktapreview.com";
+const CLIENT_ID = "OaCz3jBQxbaEnsDAFO3A";
+const REDIRECT_URI = "http://localhost:8080/";
+const AUTH_SCHEME = "OAUTH2";
+const AUTH_PARAMS = {
+        responseType: "id_token",
+        responseMode: "okta_post_message",
+        scope : [
+            "openid",
+            "email",
+            "profile",
+            "address",
+            "phone"
+        ]
+      };
+
+export default class Home extends Component {
+  static contextTypes = {
+    router: PropTypes.object
+  };
+
+  componentWillMount() {
+    const that = this;
+
+    // create Okta auth object
+    this.auth = new OktaSignIn({
+      baseUrl: BASE_URL,
+      clientId : CLIENT_ID,
+      redirectUri: REDIRECT_URI,
+      authScheme : AUTH_SCHEME,
+      authParams : AUTH_PARAMS
+    });
+
+    this.state = { authenticated : false };
+
+    setTimeout(() => {
```

Comment:
I agree but I also liked the idea that the developer get this for free in the component.